html/js/pedalboard.js

```diff
@@ -239,7 +239,7 @@ JqueryClass('pedalboard', {
         }});
 
         // Dragging the pedalboard move the view area
-        self.mousedown(function (e) {
+        self.bind('mousedown touchstart', function (e) {
```

Comment:
so `jquery.ui.touch-punch.min.js` is not working for the desktop - we have to do the equivalent our selvess - not hard.